packages/vue-router/README.md

```diff
@@ -0,0 +1,40 @@
+# `@bugsnag/vue-router-performance`
+
+> A vue router integration for BugSnag performance
+
+## Usage
+
+```
+import BugsnagPerformance from '@bugsnag/browser-performance'
+import { VueRouterRoutingProvider } from '@bugsnag/vue-router-performance'
+import { createRouter, createWebHistory } from 'vue-router'
+
+const router = createRouter({
+  history: createWebHistory('vue-router'),
+  routes: [
+    {
+      path: '/',
+      name: 'home',
+      component: HomeView
+    },
+    {
+      path: '/contacts/:contactId',
+      name: 'contact',
+      component: () => import('../views/ContactView.vue')
+    }
+  ]
+})
+
+const basename = '/vue-router'
```

Comment:
Would `/my-app` be a more realistic use for setting basename? I assume this can be omitted if it's from the root?

```suggestion
const basename = '/my-app'
```

Reply:
Also, is it significant (potential for confusion) that `createWebHistory` takes a base without a leading slash and we include it?

Reply:
`createWebHistory` accepts the leading slash. I've updated the test fixture and doc to reflect this, and renamed it base.